src/Firehose.WebCore/Properties/launchSettings.json

```diff
@@ -10,7 +10,6 @@
   "profiles": {
     "Firehose.WebCore": {
       "commandName": "Project",
-      "dotnetRunMessages": true,
```

Comment:
Not sure if this should be changed?

Reply:
Indeed, VS for Mac removes this line anytime I'm building the project.
I will fix it in my next commit.